L1Trigger/L1THGCal/src/be_algorithms/HGCalTriggerCellCalibration.cc

```diff
@@ -33,7 +33,13 @@ void HGCalTriggerCellCalibration::calibrateInMipT(l1t::HGCalTriggerCell& trgCell
     }
 
     /* correct the charge amplitude for the sensor thickness */
-    double trgCellMipP = amplitude / thickCorr_.at( cellThickness-1 );
+    double trgCellMipP = 0.;
```

Comment:
Probably better to initialize it to `amplitude`. This way the bad thickness correction doesn't destroy the energy completely, it is just a 10% effect.